### DISCUSSION TITLE
Performance on RX6600XT

### DISCUSSION THREAD

**Discussion by keibebe** [vladmandic/sdnext#3074] [General]
Hello, i need some help, im getting very bad performance using Zluda on Windows 11 with my RX6600XT. On Linux with ROCM i was able to get 4-6it/s (4-3 seconds 512x512 SD) but on windows i can only get 1.4-1.6it/s (15-13 seconds 512x512 SD). I tried enabling Deep-cache and i get up to 5it/s but quality worsens a lot and webui crashes too much.
How is Zluda performance going for you on this Graphics Card or other AMD ones?

My PC is:
R5 5600
16 GB ram 3200mhz
SSD 1 TB
RX 6600XT
Windows 11

**Comment by keibebe** [comment]
Well... I noticed that in https://github.com/brknsoul/ROCmLibs, wich are needed for cards like RX6600XT, brknsoul added a new file "Optimised_ROCmLibs_gfx1032.7z" wich when installed increased my ZLUDA SD performance to 3it/s on windows. So if you have a RX6600XT or similar try that.

**Comment by brknsoul** [comment]
For assistance with ZLUDA, join SD.Next's Discord server, specifically the [ZLUDA thread](https://discord.com/channels/1101998836328697867/1207548125841989702).
